apache-rat-core/src/main/java/org/apache/rat/report/ConfigurationReport.java

```diff
@@ -0,0 +1,285 @@
+                        Map.Entry<String, Description> entry = iter.next();
+                        if (entry.getValue().isCollection()
+                                && IHeaderMatcher.class.isAssignableFrom(entry.getValue().getChildType())) {
+                            iter.remove();
+                        }
+                    }
+                }
+                writeComment(description);
+                writer.openElement(description.getCommonName());
+                writeChildren(description, component);
+                writer.closeElement();
+                break;
+            case License:
+                writer.openElement(XMLConfigurationReader.LICENSE);
+                Description notes = null;
+                for (Description desc : description.childrenOfType(Component.Type.Parameter)) {
+                    if ("notes".equals(desc.getCommonName())) {
+                        notes = desc;
+                    } else {
+                        writeAttribute(desc, component);
+                    }
+                }
+                for (Description desc : description.childrenOfType(Component.Type.BuilderParam)) {
+                    if ("family".equals(desc.getCommonName())) {
+                        try {
+                            ILicenseFamily family = (ILicenseFamily) desc.getter(component.getClass())
+                                    .invoke(component);
+                            if (family != null) {
+                                writer.attribute("family", family.getFamilyCategory().trim());
+                            }
+                        } catch (NoSuchMethodException | IllegalAccessException | IllegalArgumentException
+                                | InvocationTargetException | SecurityException e) {
+                            configuration.getLog().error(e.toString());
+                        }
+                    }
+                }
+                // end of attributes
+                if (notes != null && StringUtils.isNotBlank(notes.getParamValue(configuration.getLog(), component))) {
+                    writeComment(notes);
+                    writer.openElement(notes.getCommonName());
+                    writeContent(notes, component);
+                    writer.closeElement();
+                }
+                for (Description desc : description.childrenOfType(Component.Type.Unlabeled)) {
+                    writeDescription(desc, component);
+                }
+                writer.closeElement();
+                break;
+            case Parameter:
+                if ("id".equals(description.getCommonName())) {
+                    try {
+                        String idValue = description.getParamValue(configuration.getLog(), component);
+                        // if a UUID skip it.
+                        if (idValue != null) {
+                            UUID.fromString(idValue.toString());
+                            return;
```

Comment:
Does this leave broken XML? Maybe I do not understand it correctly ..... or does it just not write any thing as parameters do not have descriptions?